test/Deployment/Test_DeploymentManagerAutoInitialize/CPP/Test_DeploymentManagerAutoInitialize_CPP_Default/Test_DeploymentManagerAutoInitialize_CPP_Default.vcxproj

```diff
@@ -38,7 +38,7 @@
   <PropertyGroup>
     <ConfigurationType>DynamicLibrary</ConfigurationType>
     <UseOfMfc>false</UseOfMfc>
-    <PlatformToolset>v142</PlatformToolset>
```

Comment:
With this PR there's no "v142" references in the repo?

Reply:
![image](https://user-images.githubusercontent.com/48363984/186765261-f4378c84-3998-40fb-baf8-898bdafd923a.png)
 None found